test/message/test_runner_output.out

```diff
@@ -139,13 +139,45 @@ not ok 13 - async assertion fail
     *
     *
   ...
+# Subtest: async assertion object fail
+not ok 14 - async assertion object fail
+  ---
+  duration_ms: *
+  failureType: 'testCodeFailure'
+  error: |-
+    Expected values to be loosely deep-equal:
+    
+    {
+      bar: 1,
+      foo: 1
+    }
+    
+    should loosely deep-equal
+    
+    {
+      bar: 2
+    }
+  code: 'ERR_ASSERTION'
+  name: 'AssertionError'
+  expected: { bar: 2 }
+  actual: { foo: 1, bar: 1 }
```

Comment:
What will happen if these are large objects?

Reply:
a large string would appear I guess. why is that a problem?

Reply:
Technically, probably nothing. It's just not a great experience to have to scroll horizontally.

Reply:
frankly, I have no strong opinion on how this issue should be fixed, so if there is an approach you think we should take, then let's take it.
I just haven't understood what you are suggesting and what would be a context object

Reply:
I think if possible, we should convert the object to YAML vertically instead of stringifying it horizontally. I plugged the following object into https://www.convertsimple.com/convert-javascript-object-to-yaml/:

```js
{
  actual: { foo: 1 },
  expected: { foo: 2 }
}
```

It generates this:

```yaml
actual:
  foo: 1
expected:
  foo: 2
```

That seems more user friendly to read. The problem is that it's more complex to implement. I won't ask you to do a lot more work to implement that solution, but I think it's at least worth calling out as something that could be improved.

Reply:
got it.